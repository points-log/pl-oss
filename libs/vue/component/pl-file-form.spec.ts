import { shallowMount } from '@vue/test-utils';
import Vue from 'vue';
import Vuetify from 'vuetify';
import PLFileForm from './pl-file-form.vue';

Vue.use(Vuetify);

describe('PLFileForm', () => {
  it('should match the snapshot', () => {
    const stubs = { 'pl-cancel-btn': true, 'pl-done-btn': true };
    const propsData = { title: 'Title' };
    const wrapper = shallowMount(PLFileForm, { propsData, stubs });
    expect(wrapper.element).toMatchSnapshot();
  });
});
