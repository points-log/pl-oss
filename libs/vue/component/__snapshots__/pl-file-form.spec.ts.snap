// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLFileForm should match the snapshot 1`] = `
<v-form-stub>
  <v-card-stub
    loaderheight="4"
    tag="div"
  >
    <v-card-title-stub>
      Title
    </v-card-title-stub>
    <v-card-text-stub>
      <v-file-input-stub
        backgroundcolor=""
        clearable="true"
        clearicon="$clear"
        countersizestring="$vuetify.fileInput.counterSize"
        counterstring="$vuetify.fileInput.counter"
        errorcount="1"
        errormessages=""
        hidedetails="auto"
        loaderheight="2"
        messages=""
        outlined="true"
        prependicon="$file"
        rules=""
        successmessages=""
        truncatelength="22"
        type="file"
      />
    </v-card-text-stub>
    <v-card-actions-stub>
      <v-spacer-stub />
      <pl-cancel-btn-stub />
      <pl-done-btn-stub />
    </v-card-actions-stub>
  </v-card-stub>
</v-form-stub>
`;
